docs/user-guide/api-mediation/configuration-saf-resource-checking.md

```diff
@@ -96,8 +106,8 @@ The following YAML presents the structure of the file:
 ```
 
 :::note Notes
-- Classes and resources are mapped into a map, user IDs into a list.
-- The load method does not support formatting with dots, such as shown in the following example:
+- Classes and resources are mapped into a map with user IDs contained in a list.
+- The load method does not support formatting with periods `.`, such as shown in the following example:
   **Example:** `{CLASS}.{RESOURCE}`
   Ensure that each element is separated.
 - The field `safAccess` is not required to define an empty file without a definition.
```

Comment:
Can you please add a conclusion statement that describes the end result of completing this procedure? Thanks